obs/preinstallimage-bios.sh.in

```diff
@@ -680,6 +680,12 @@ rm /etc/init.d/ssh*
 echo "UseDNS no" >> /etc/ssh/sshd_config
 # Limit to Administrator user (i.e. no Viewers like "monitor")
 echo "AllowGroups bios-admin root" >> /etc/ssh/sshd_config
+
+# Remove host saving when using ssh from IPM2 to another machine
+echo "StrictHostKeyChecking no" >> /etc/ssh/sshd_config
```

Comment:
Thank you for breaking our ssh server. These lines belong in ssh client config.

````
Nov 02 13:29:33 fty-devel sshd[5823]: /etc/ssh/sshd_config: line 126: Bad configuration option: StrictHostKeyChecking
Nov 02 13:29:33 fty-devel sshd[5823]: /etc/ssh/sshd_config: line 127: Bad configuration option: UserKnownHostsFile
Nov 02 13:29:33 fty-devel sshd[5823]: /etc/ssh/sshd_config: terminating, 2 bad configuration options
````

Reply:
Also, calling with such options should be done on command line of the client that "really needs" to bypass security (e.g. because remote monitored systems do change), we have dozens of examples in codebase (ssh -o K=V ...) so there is no reason to make a mess system-wide.

But wait, isn't the point of saved SSH fingerprints to confirm that we are still talking to the same server at some remote IP, so if that identification changed - it is generally a security issue and a human should reconfirm that now we want a new key trusted there? (Or have an option to bypass that - again, at responsibility of the human)

Reply:
Thanks Jim, I'm already fixing it